content/en/code_analysis/software_composition_analysis/_index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Software Composition Analysis
+kind: documentation
+description: Learn about Datadog Software Composition Analysis to scan your imported open-source libraries for known security vulnerabilities before you ship to production.
+is_beta: true
+further_reading:
+- link: "https://www.datadoghq.com/blog/iast-datadog-application-vulnerability-management/"
+  tag: "Blog"
+  text: "Enhance application security in production with Datadog Application Vulnerability Management"
+- link: "/getting_started/application_security/software_composition_analysis"
+  tag: "Documentation"
+  text: "Getting Started with Software Composition Analysis"
+- link: "/security/application_security/software_composition_analysis/"
+  tag: "Documentation"
+  text: "Learn about Software Composition Analysis"
+- link: "/integrations/guide/source-code-integration/"
+  tag: "Documentation"
+  text: "Learn about the Source Code Integration"
+---
+
+{{% site-region region="gov" %}}
+<div class="alert alert-danger">
+    Code Analysis is not available for the {{< region-param key="dd_site_name" >}} site.
+</div>
+{{% /site-region %}}
+
+{{< callout url="#" btn_hidden="true" header="Try the Beta!" >}}
+Code Analysis is in public beta.
+{{< /callout >}}
+
+## Overview
+
+Software Composition Analysis (SCA) scans open source libraries imported into repositories through package managers such as `npm` for [known vulnerabilities][1]. SCA enables engineering teams to identify vulnerable libraries early on in the development life cycle so they can update them to non-vulnerable versions or remove them entirely to ensure their production codebase is secure.
+
+SCA can run in CI pipelines by using [Code Analysis][3] and provide runtime monitoring capabilities by using [Datadog Application Security][1].
+
+For more information, see the [Application Security documentation][4].
+
+## Languages
+
+SCA currently supports scanning the following languages and technologies for vulnerable libraries:
+
+1. Go
+2. Java
+3. NodeJS
+4. Python
+5. Ruby
```

Comment:
```suggestion
- Ruby
```
Maybe personal preference but I think it reads better with dashes instead of numbers.